devel/libecl/src/smspec_node.c

```diff
@@ -736,6 +736,38 @@ smspec_node_type * smspec_node_alloc_lgr( ecl_smspec_var_type var_type ,
   }
 }
 
+smspec_node_type* smspec_node_copy( const smspec_node_type* node ) {
```

Comment:
The ert code has as a convention that: "every single function which allocates memory should have *alloc* as part of the name" - so cool if you rename:  `smspec_node_alloc_copy( )`.